.devcontainer/scripts/2_post_requirements.sh

```diff
@@ -0,0 +1,19 @@
+# This script runs after installing the pip requirements but before the Docker container has finished building
+
+# Install session manager plugin for aws cli
+curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_64bit/session-manager-plugin.deb" -o "session-manager-plugin.deb";
+sudo dpkg -i session-manager-plugin.deb;
+rm session-manager-plugin.deb;
+
+# Install SQL Server ODBC driver
+curl https://packages.microsoft.com/keys/microsoft.asc | sudo tee /etc/apt/trusted.gpg.d/microsoft.asc
+
+curl https://packages.microsoft.com/config/debian/11/prod.list | sudo tee /etc/apt/sources.list.d/mssql-release.list
+
+sudo apt-get update
+sudo ACCEPT_EULA=Y apt-get install -y msodbcsql17
+sudo ACCEPT_EULA=Y apt-get install -y mssql-tools
```

Comment:
What are these EULAs?

Reply:
Essentially they're about not modifying or re-distributing the tools. The tools are fine to install and use as-is for development.